### PR TITLE
Add documentation about the bootstrapping in AL2

### PR DESCRIPTION
**Pull request by martina-if**
Do you think this is useful to keep here?

- [x] Added labels for change area (e.g. `area/nodegroup`), target version (e.g. `version/0.12.0`) and kind (e.g. `kind/improvement`)
- [x] Make sure the title of the PR is a good description that can go into the release notes

### CODE REVIEW

**Comment by michaelbeaumont** [comment]
I'm undecided on whether it should be visible on the website but I like the docs. I just remembered [docs/current-001-node_bootstrap.md](https://github.com/weaveworks/eksctl/blob/master/docs/current-001-node_bootstrap.md), do you think they should be merged?

File: docs/current-001-node_bootstrap.md [hunk 1]
```diff
@@ -113,42 +164,158 @@ ExecStart=/usr/bin/kubelet \
   --network-plugin=cni \
   --cni-bin-dir=/opt/cni/bin \
   --cni-conf-dir=/etc/cni/net.d \
-  --pod-infra-container-image=${AWS_EKS_ECR_ACCOUNT}.dkr.ecr.${AWS_DEFAULT_REGION}.amazonaws.com/eks/pause-amd64:3.1 \
-  --kubeconfig=/etc/eksctl/kubeconfig.yaml \
-  --config=/etc/eksctl/kubelet.yaml
-```
+  --pod-infra-container-image=${AWS_EKS_ECR_ACCOUNT}.dkr.ecr.${AWS_DEFAULT_REGION}.${AWS_SERVICES_DOMAIN}/eks/pause:3.1-eksbuild.1 \
+  --kubeconfig=<a href="#kubeconfig">/etc/eksctl/kubeconfig.yaml</a> \
```
Comment (michaelbeaumont) [inline docs/current-001-node_bootstrap.md:168]: some HTML tags snuck in here

Comment (martina-if) [reply docs/current-001-node_bootstrap.md:168]: That's on purpose! :) I thought adding that to navigate through the different files might be helpful to understand. What do you think?

File: docs/current-001-node_bootstrap.md [hunk 2]
```diff
@@ -61,48 +61,99 @@ Summary:
...
+
+NODE_IP="$(get_metadata local-ipv4)"
+INSTANCE_ID="$(get_metadata instance-id)"
+INSTANCE_TYPE="$(get_metadata instance-type)"
+AWS_SERVICES_DOMAIN="$(get_metadata services/domain)"
+
+
+source <a href="#kubelet-environment-variables">/etc/eksctl/kubelet.env</a> # this can override MAX_PODS
+
+cat &gt; <a href="#kubelet-local-environment-variables">/etc/eksctl/kubelet.local.env></a>  &lt;&lt;EOF
```
Comment (michaelbeaumont) [inline docs/current-001-node_bootstrap.md:124]: here one `>` too many before `</a>`

File: docs/current-001-node_bootstrap.md (hunk 1 above)
Comment (michaelbeaumont) [reply docs/current-001-node_bootstrap.md:168]: oh cool I didn't realize it works inside \`\`!

File: docs/current-001-node_bootstrap.md (hunk 2 above)
Comment (martina-if) [reply docs/current-001-node_bootstrap.md:124]: thanks! good catch

**Review by michaelbeaumont — APPROVED** [review]

File: docs/current-001-node_bootstrap.md (hunk 1 above)
Comment (martina-if) [reply docs/current-001-node_bootstrap.md:168]: They only work inside `<pre></pre>` tags, but that makes the block look the same in GH (not in the website though).

**Review by michaelbeaumont — APPROVED** [review]

**Pull request merged**